browser/ntp_background_images/BUILD.gn

```diff
@@ -10,6 +10,8 @@ assert(enable_custom_background)
 source_set("ntp_custom_background_images") {
```

Comment:
I know this is pre-existing, but this target is misnamed. It should be ntp_background_images and the buildflag should be enable_ntp_background_images. Or alternatively everything should be named ntp_custom_background_images. The inconsistent naming is confusing.

Reply:
Yeah, I was too lazy to rename this directory 😅 I guess why it was name "ntp_background_image" is that this was to provide delegate implementation for `//components/ntp_background_image'.

The way I see it is 'NTPBackground can be one of 1) brave image 2) custom image 3)  solid color(WIP) 4) gradient color(WIP) .  So I named NTPBackgroundPref as is as it covers all these options. And I'd like to suggest this directory name be "ntp_background" as it provides more than just 'images'.